action.yml

```diff
@@ -0,0 +1,58 @@
+name: "Deploy Beanstalk Application"
+author: "Marcel Arns <marcel.arns@moneymeets.com>"
+description: "Create Beanstalk application version and update environment"
+inputs:
+  aws_access_key_id:
+    description: "AWS Access Key"
+    required: true
+  aws_secret_access_key:
+    description: "AWS Secret Access Key"
+    required: true
+  aws_region:
+    description: "AWS Region"
+    required: false
+    default: "eu-central-1"
+  version_label:
+    description: "Version label (default: github.sha)"
+    required: false
+    default: ${{ github.sha }}
+  version_description:
+    description: "Version description (default: github.run_number)"
+    required: false
+    default: "GitHub Action #${{ github.run_number }}"
+  build_and_upload_image:
+    description: "Build and upload the docker image"
+    required: true
+  wait_for_deployment:
+    description: "Whether the action should wait for the deployment to finish and log status messages during wait"
+    required: false
+    default: "true"
```

Comment:
```suggestion
    default: true
```

Muss das ein string sein?

Reply:
Laut @marns93 leider ja 😞  die Github-CI-Typen packen alles in Environment-Variablen...

Reply:
![image](https://user-images.githubusercontent.com/607401/107820037-6e5da280-6d7a-11eb-994e-fcad104573c6.png)

Reply:
Ja, muss leider ein `string` sein. :(